File: malevich-web/src/app/_transfer/tradeHistoryDto.ts
import {ArtworkStockDto} from "./artworkStockDto";
import {OrderDto} from "./orderDto";

export class TradeHistoryDto {
  id: number;
  artworkStock: ArtworkStockDto;
  effectiveDate: Date;
  askOrder: OrderDto;
  bidOrder: OrderDto;
  amount: number;
  quantity: number;
}

File: malevich-server/src/main/java/io/malevich/server/transfer/TradeHistoryDto.java
package io.malevich.server.transfer;

import lombok.Getter;
import lombok.Setter;

import java.sql.Timestamp;


@Getter
@Setter
public class TradeHistoryDto {


    private Long id;

    private ArtworkStockDto artworkStock;

    private Timestamp effectiveDate;

    private OrderDto askOrder;

    private OrderDto bidOrder;

    private Double amount;

    private Long quantity;

}
